PrxDecrypter.cpp

```diff
@@ -829,7 +856,7 @@ static int pspDecryptType2(const u8 *inbuf, u8 *outbuf, u32 size)
 	// check if range is non-zero
 	if (std::any_of(inbuf+0xD4, inbuf+0xD4+0x58, [](u8 x) { return x != 0; }))
 	{
-		return -2;
+		//return -2;
```

Comment:
Can you add a comment about why we shouldn't return in this case (and the cases below)?

Reply:
Indeed, I thought about explaining it at some point then forgot.. I added an explanation. :)